public/index.html

```diff
@@ -2,15 +2,35 @@
 <html lang="cs">
 	<head>
 		<meta charset="utf-8" />
-		<link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		<meta name="theme-color" content="#264653" />
 		<meta
 			name="description"
 			content="Web application that connects programmers based on their experience, interests and the type of person they are looking for (mentor, buddy, mentee) through matching and contact sharing."
 		/>
-		<!-- <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" /> -->
-		<link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```

Comment:
Why did you remove this default manifest? :thinking:

You technically did a copy in the `site.webmanifest`, so now we have 2 very similar manifests and only linking one. :smile:

Reply:
Zkopíroval jsem kód vygenerovaný faviconou. Takže jsem si myslel, že webmanifest je součástí favicony. Měl bych tedy odstranit webmanifest a nechat tam jen manifest.json?

![image](https://user-images.githubusercontent.com/58363173/109422532-3726fc80-79dc-11eb-9c90-421daeee88be.png)

Reply:
Yeah, I would leave the default one, because they are almost same. :+1:

Reply:
ok, fixed 👍